### PR TITLE
Show 3 decimals in bank value when bank value >= 1B

### PR DESCRIPTION
**Pull request by Ameobea**
* This allows users to feel a better sense of progression when farming and more precisely judge the impact of trades on their net worth.

### CODE REVIEW

**Comment by gc** [comment]
This should be a standalone function in the util folder

**Comment by Ameobea** [comment]
ok done.

File: src/lib/util/formatBankValue.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { toKMB } from 'oldschooljs/dist/util/util';
+
+const formatBankValue = (totalValue: number): string => {
+	if (totalValue < 1e9) {
```
Comment (gc) [inline src/lib/util/formatBankValue.ts]: Use `const billion = 1_000_000_000;` instead (declared outside the function), for readability purposes

**Comment by gc** [comment]
Closing as its been 2 weeks with no updates. Feel free to re-open.

**Comment by Ameobea** [comment]
Srry, missed the feedback notification.

I've updated my branch with the requested change.  If you want, I can open another PR, or you could re-open this one.

**Pull request closed without merging**